SAP/fundamental
# #1129 builds an IE11 compatible dist file

Closes sap/fundamental#1129

Updates dist build to output an IE11 compatible version of the full library. **Any product that needs to support IE11 MUST use this version.**

#### Test

* Run `gulp build:dist --production` and see a reg and min version of the library with IE11 suffix
* View the compiled files to see fallback values for CSS vars

```
body {
  margin: 0;
  background-color: #edeef0;
  background-color: var(--fd-background-color);
  -webkit-font-smoothing: antialiased;
  color: #32363a;
  color: var(--fd-color)
}
```

#### Changelog

**New**

* Add `all-ie11.scss` file which sets the `$fd-support-css-var-fallback;` fallback flag to `true`

**Changed**

* Modifies build task to handle properly name the IE 11 files

**Removed**

* N/A
